tests/framework/installer/install_helper.go

```diff
@@ -253,6 +275,13 @@ func (h *InstallHelper) InstallRookOnK8sWithHostPathAndDevices(namespace, storeT
 		}
 	} else {
 		onamespace = SystemNamespace(namespace)
+
+		if h.k8shelper.VersionAtLeast("v1.7.0") {
```

Comment:
this if/else block doesn't look very useful, forgot to remove it during refactoring?

Reply:
Good catch on that... forgot to remove. Removing